### ISSUE TITLE
Ranking Question - Drag & Drop works on disabled ("Editable if" -> "False") choices

### ISSUE THREAD

**Issue by GER-Jani** [surveyjs/survey-library#6834]
### Are you requesting a feature, reporting a bug or asking a question?
Bug report

### What is the current behavior?
Using ranking including "selectToRank", a disabled (`"enableIf": "False"`) choice item can be dragged & dropped and the required question can be submitted.

### What is the expected behavior?
Since the choice item is disabled, it sould not be selectable or be "drag & dropable".

### How would you reproduce the current behavior (if this is a bug)?
Create a new survey and use this code:

```
{
 "logoPosition": "right",
 "pages": [
  {
   "name": "page1",
   "elements": [
    {
     "type": "ranking",
     "name": "Test",
     "title": "Drag & Drop Test",
     "isRequired": true,
     "choices": [
      "Yes, Drag & Drop should work",
      {
       "value": "Nope, Drag & Drop should not work !!",
       "enableIf": "False"
      },
      "Yes, Drag & Drop should work here again"
     ],
     "selectToRankEnabled": true,
     "selectToRankAreasLayout": "vertical"
    }
   ]
  }
 ]
}
```

Demo: https://surveyjs.io/published?id=9fa95c22-e91d-4785-9a9a-c039b512ca30

Simple real world example: https://surveyjs.io/published?id=e90bb4d8-8a15-4c89-a446-c4525296febc

### Specify your

* browser: Google Chrome
* browser version: Version 116.0.5845.111
* surveyjs platform (angular or react or jquery or knockout or vue): SurveyJS website
* surveyjs version: Current

**Comment by JaneSjs** [comment]
Hi @GER-Jani,
Thank you for reporting this issue. However, it is possible to rank a disabled item regardless of the `selectToRankEnabled` option. It seems that the item is not actually disabled. I forwarded this issue to our developers for further investigation. We'll update you once we have any news to report.

Thanks

**Comment by JaneSjs** [comment]
@GER-Jani,
To consider the most suitable option, I need to elaborate on the usage scenario behind this issue. Would you please clarify the purpose of disabling ranking items? What is the usage scenario? Do you wish to conditionally enable/disable items to allow respondents order only certain items? 

Please describe the use case so that we can clearly understand the overall usage scenario.

Thanks

**Comment by GER-Jani** [comment]
@JaneSjs Thank you for confirming and forwarding this bug.


I have already attached a demo that illustrates the use case for one of our customer projects:

> Simple real world example: https://surveyjs.io/published?id=e90bb4d8-8a15-4c89-a446-c4525296febc

Personally, I would have realized that kind of survey behaviour via radiogroups, but our customer prefers the option using drag & drop ...
An alternative would be to simply hide (`visibleIf`) the corresponding choice option (the option that should be disabled here), which works well.
But our customer argues that this will probably confuse users if choice options are suddenly missing ...

**Comment by JaneSjs** [comment]
Thank you for the clarification, @GER-Jani. We'll take this note into consideration. Please stay tuned to our updates.
